### PR TITLE
Adds the Remand Received action to MDR & MandateHold Tasks

### PR DESCRIPTION
**Pull request by lomky**
Bumps [CASEFLOW-98](https://vajira.max.gov/browse/CASEFLOW-98)

### Description
Adds the new Task Action `mandate received` and makes it available on the MDR task.

Stack PR for [CASEFLOW-98](https://vajira.max.gov/browse/CASEFLOW-98)

Part 1 - Adds the Remand Received action to MDR Task
Part 2 - TK
Part 3 - TK

### Acceptance Criteria
 - [ ] Work is behind the feature toggle: mdr_cavc_remand
 - [ ] Accessible to the CAVC Litigation Support team
 - [ ] MDR task has the `Remand Ready for 90 Day Letter` action
 - [ ] MandateHold task has the `Ready for VLJ` action
 - [ ] Both of these actions show the Add Mandate modal  

### Testing Plan
1. Create an MDR cavc remand
    `FactoryBot.create(:cavc_remand, :mdr).remand_appeal.uuid`
2. Sign in as a CAVC Lit Support user and nagivate to the appeal
   - [x] Confirm the action `Remand ready for 90-day letter`
   - [x] Confirm selection the action shows the entry modal
3. that's it for PR 1! submission is PR 2
4. Repeat for No mandate Straight Reversal
    `FactoryBot.create(:cavc_remand, :straight_reversal, :no_mandate).remand_appeal.uuid`
   - [ ] Confirm the action `Ready for VLJ`
   - [ ] Confirm selection the action shows the entry modal
5. Repeat for No mandate Death Dismissal
   `FactoryBot.create(:cavc_remand, :death_dismissal, :no_mandate).remand_appeal.uuid`
   - [ ] Confirm the action `Ready for VLJ`
   - [ ] Confirm selection the action shows the entry modal

### User Facing Changes
 - [ ] Screenshots of UI changes added to PR & Original Issue

 BEFORE|AFTER
 ---|---
MDR | MDR case details page showing new action
Straight Reversal & Death Dismissal | Death Dismissal case details page showing new action

### CODE REVIEW

File: app/models/tasks/mandate_hold_task.rb [hunk 1]
```diff
@@ -40,22 +40,13 @@ def default_instructions
 
   # Actions for both admins and non-admins
   TASK_ACTIONS = [
-    Constants.TASK_ACTIONS.TOGGLE_TIMED_HOLD.to_h
+    Constants.TASK_ACTIONS.TOGGLE_TIMED_HOLD.to_h,
+    Constants.TASK_ACTIONS.CAVC_REMAND_RECEIVED_VLJ.to_h
   ].freeze
```
Comment (lomky) [inline app/models/tasks/mandate_hold_task.rb:45]: CAVCLitSupport doesnt want distinct admin/user actions

**Review by n1zyy — APPROVED** [review]
Code Climate is angry about the actions refactor, but otherwise the code looks good and running through it in the UI works.

**Comment by n1zyy** [comment]
Screen Shot 2021-02-04 at 5 33 46 PM

**Comment by lomky** [comment]
With the newly appended instructions
Caseflow Case Details page, CAVC Remand details section, showing instructions from both creation and mandate

**Pull request merged**